src/libmongoc/doc/mongoc_client_encryption_create_datakey.rst

```diff
@@ -16,7 +16,25 @@ Synopsis
       bson_value_t *keyid,
       bson_error_t *error);
 
-Alias function equivalent to :symbol:`mongoc_client_encryption_create_key`.
+Creates a new key document in the key vault collection and sets ``keyid`` to the UUID of the newly created key if ``keyid`` is not NULL. The new key can be used to configure automatic encryption (see :symbol:`mongoc_client_enable_auto_encryption()` and :symbol:`mongoc_client_pool_enable_auto_encryption()`) or for explicit encryption (see :symbol:`mongoc_client_encryption_encrypt()`).
+
+The created key document is inserted into the key vault collection (identified via :symbol:`mongoc_client_encryption_opts_set_keyvault_namespace()`) with majority write concern.
+
+``keyid`` is always initialized (even on error). Caller must call :symbol:`bson_value_destroy()` on ``keyid`` to free.
+
+Parameters
+----------
+
+* ``client_encryption``: A :symbol:`mongoc_client_encryption_t`.
+* ``kms_provider``: A string identifying the Key Management Service (KMS) provider used to encrypt the datakey (e.g. "aws" or "local").
+* ``opts``: A :symbol:`mongoc_client_encryption_datakey_opts_t`
```

Comment:
The fact that the option struct itself was named `datakey_opts_t` makes me realize that it would have been a mistake to have kept `create_key` around. In `#drivers-fle`, there was some discussion about `create_key` being a generic method that could later create other types of keys -- but that wouldn't have been possible if its libmongoc API was restricted to `datakey_opts_t`.